system/ui/onroad/augmented_road_view.py

```diff
@@ -0,0 +1,119 @@
+import pyray as rl
+import numpy as np
+from openpilot.system.ui.widgets.cameraview import CameraView
+from msgq.visionipc import VisionStreamType
+from openpilot.system.ui.lib.application import gui_app
+from cereal import messaging
+
+
+VIEW_FROM_DEVICE = np.array([
+    [0.0, 1.0, 0.0],
+    [0.0, 0.0, 1.0],
+    [1.0, 0.0, 0.0]
+])
+
+FCAM_INTRINSIC_MATRIX = np.array([
```

Comment:
now that it's in python, you should be able to import this from `openpilot.common`